src/helpers/highlight.ts

```diff
@@ -20,8 +20,22 @@ export default function highlight({
   hit,
   cssClasses = {},
 }: HighlightOptions): string {
-  const { value: attributeValue = '' } =
-    getPropertyByPath(hit._highlightResult, attribute) || {};
+  const highlightAttributeResult = getPropertyByPath(
+    hit._highlightResult,
+    attribute
+  );
+
+  // @MAJOR fallback to attribute value if highlight is not found
+  warning(
+    highlightAttributeResult,
+    `Could not enable highlight for "${attribute}", will display an empty string.
+Please check whether this attribute exists and is either searchable or specified in \`attributesToHighlight\`.
+
+See: https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/highlighting-snippeting/js/
```

Comment:
I see other examples using an "alg.li" URL shortener, but I didn't find documentation on how to access it. If you have an idea on how to do it, I'll happily update the links.

Reply:
I just created https://alg.li/highlighting (it's in short.cm, you can DM if you want access too, or check Okta)

Reply:
```suggestion
See: https://alg.li/highlighting
```

Reply:
Replaced all occurrences in 432be6e.